Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelCalculationImpl.kt

```diff
@@ -0,0 +1,66 @@
+package de.rki.coronawarnapp.risk
+
+import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import de.rki.coronawarnapp.server.protocols.ApplicationConfigurationOuterClass
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.math.round
+
+@Singleton
+class RiskLevelCalculationImpl @Inject constructor() : RiskLevelCalculation {
```

Comment:
With the idea that due to the interface we might replace the implementation or offer alternatives, should we maybe name this `DefaultRiskLevelCalculation`?